src/python/nimbusml/internal/entrypoints/transforms_featureselectorbymutualinformation.py

```diff
@@ -55,7 +55,7 @@ def transforms_featureselectorbymutualinformation(
             none_acceptable=False,
             is_of_type=str)
     if label_column_name is not None:
-        inputs['LabelColumn'] = try_set(
+        inputs['LabelColumnName'] = try_set(
```

Comment:
Is the `"PassAs": "LabelColumn",` in manifest.json not working as expected for this field?